### PR TITLE
add test for demisto docker img

### PR DESCRIPTION
**Pull request by esharf**
## Status
Ready.

## Related Issues
fixes: https://github.com/demisto/etc/issues/21905

## Description
added a test if the docker img is a demisto docker img.

## Screenshots
new error
new_error
old error screenshot existed in the issue.

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/docker.py [hunk 1]
```diff
@@ -195,6 +197,9 @@ def get_docker_image_latest_tag(docker_image_name, yml_docker_image):
         Returns:
             The last updated docker image tag
         """
+        if yml_docker_image and not yml_docker_image.startswith('demisto/'):
+            print_error('docker img must be a demisto docker img.')
```
Comment (bakatzir) [inline demisto_sdk/commands/common/hook_validations/docker.py]:
```suggestion
            print_error('docker image must be a demisto docker image. e.g: demisto/python:<tag>')
```

File: demisto_sdk/commands/common/hook_validations/docker.py [hunk 2]
```diff
@@ -195,6 +197,9 @@ def get_docker_image_latest_tag(docker_image_name, yml_docker_image):
         Returns:
             The last updated docker image tag
         """
+        if yml_docker_image and not yml_docker_image.startswith('demisto/'):
```
Comment (bakatzir) [inline demisto_sdk/commands/common/hook_validations/docker.py]: can be demisto or demistodev

File: demisto_sdk/commands/common/hook_validations/docker.py [hunk 3]
```diff
@@ -195,6 +197,12 @@ def get_docker_image_latest_tag(docker_image_name, yml_docker_image):
         Returns:
             The last updated docker image tag
         """
+        if yml_docker_image:
+            if yml_docker_image.startswith('devdemisto/'):
+                print_warning('docker image must be a demisto docker image. e.g: demisto/python:<tag>')
```
Comment (bakatzir) [inline demisto_sdk/commands/common/hook_validations/docker.py]:
```suggestion
                print_warning('docker image must be a demisto docker image. When the docker image is ready, please rename it to: demisto/python:<tag>')
```

File: demisto_sdk/commands/common/tests/docker_test.py [hunk 4]
```diff
@@ -64,6 +64,18 @@ def test_get_docker_image_latest_tag(image):
     assert int(tag.split('.')[3]) >= 2728
 
 
+data_test_none_demisto_docker = [
+    (('demisto/algorithmia', '1.0.0.6016'), '1.0.0.6016'),
+    (('demisto/google-api-py3', '1.0.0.5992'), '1.0.0.5992'),
+    (('blabla/google-api-py3', '1.0.0.5992'), ''),
+]
```
Comment (bakatzir) [inline demisto_sdk/commands/common/tests/docker_test.py]: please add a test for devdemisto as well

**Comment by Itay4** [comment]
@bakatzir @esharf can we merge this?

**Pull request merged**